packages/pigeon/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## NEXT
+## 1.0.11
 
 * [ci] Started transitioning to a Dart test runner, added windows support.
+* [front-end] Started issuing an error if enums are used in type arguments.
+* [front-end] Passed through all enums, referenced or not so they can be used as
+  a work around for direct enum support.
```

Comment:
Nit: There's now [a style guide for CHANGELOGs](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style), so these should follow it (e.g., Starts issuing, Passes through)

Reply:
Done